src/gl/camera-shader.cpp

```diff
@@ -124,7 +123,6 @@ namespace librealsense
         {
             camera_mesh.push_back(load_model(uncompress_d415_obj));
             camera_mesh.push_back(load_model(uncompress_d435_obj));
-            camera_mesh.push_back(load_model(uncompress_L515_obj));
```

Comment:
You don't have latest code:

On this file please also change this:
```
if (starts_with(dev_name, "Intel RealSense L5")) index = 2;
                if (starts_with(dev_name, "Intel RealSense D45")) index = 3;
```

to this:
```
    if (starts_with(dev_name, "Intel RealSense D45")) index = 2;
```

Reply:
Right, fixed.